staging/src/k8s.io/csi-translation-lib/plugins/in_tree_volume.go

```diff
@@ -165,6 +229,126 @@ func translateTopology(pv *v1.PersistentVolume, topologyKey string) error {
 	return nil
 }
 
+// getTopologyLabel checks if the kubernetes topology used in this PV is GA
+// and return the zone/region label used.
+// It first check the NodeAffinity to find topology. If nothing is found,
+// It checks the PV labels. If it is empty in both places, it will return
+// GA label by default
+func getTopologyLabel(pv *v1.PersistentVolume) (bool, string, string) {
```

Comment:
No. I might just remove it. I first made it and thought it might get handy somehow. But since it is not used anywhere. Guess we dont need this extra bool